### DISCUSSION TITLE
Abnormal Values of Environment Temperature

### DISCUSSION THREAD

**Discussion by USTCEarthDefense** [PaddlePaddle/PaddleSpatial#177] [BaiduKDDCup2022] [answered]
Hi, 

In checking the data set, I found there are tons of abnormal values of environment temperature. For example, as the data of turbine 121 shown below. Will the organizer also consider records with such abnormal env temp invalid for evaluation? 
![image](https://user-images.githubusercontent.com/50040116/173942419-484f02c5-5d24-4014-8364-918f136b12a8.png)

**Comment by zhoujingbo** [comment, marked as answer]
Hi,

Abnormal temperature does not directly result in an invalid Patv value. There are two reasons: 1) there is no standard rule to set the threshold for abnormal temperature values; 2) the abnormal value of temperature may be caused by the damage of temperature sensors, but the turbine can still work normally.

 Therefore, we still take such records for evaluations.  Whether to use such values to train the model depends on your decision. 

Thanks a lot for your support.

Regards
